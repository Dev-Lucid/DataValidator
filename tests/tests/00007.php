<?php
$field = dvr::field('field1')->rule('email','fail_message');

$out  = '';
list($result,$errors) = $field->test(array('field1'=>'test@test.com'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'test@test'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'@test.com'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'testtest.com'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'test@@test.com'));
$out .= ($result)?'PASSES':'FAIL';

file_put_contents($output_path,$out);
?>