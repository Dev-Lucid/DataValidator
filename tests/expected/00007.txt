PASSES
FAIL
FAIL
FAIL
FAIL